extensions/amp-accordion/amp-accordion.md

```diff
@@ -29,7 +29,7 @@ limitations under the License.
 <table>
   <tr>
     <td class="col-fourty"><strong>Description</strong></td>
-    <td>A clickable stacked list of headers that reveal or hide content associated with them.</td>
+    <td>A clickable stacked list of headers that collapse or expand content sections.  associated with them.</td>
```

Comment:
should this be removed as well? `  associated with them.`

Reply:
oops, my bad, removed!